facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerResultProcessor.java

```diff
@@ -0,0 +1,199 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.facade.deliverables;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+import org.jboss.pnc.api.deliverablesanalyzer.dto.Artifact;
+import org.jboss.pnc.api.deliverablesanalyzer.dto.ArtifactType;
+import org.jboss.pnc.api.deliverablesanalyzer.dto.Build;
+import org.jboss.pnc.api.deliverablesanalyzer.dto.MavenArtifact;
+import org.jboss.pnc.api.deliverablesanalyzer.dto.NPMArtifact;
+import org.jboss.pnc.enums.ArtifactQuality;
+import org.jboss.pnc.enums.RepositoryType;
+import org.jboss.pnc.model.ProductMilestone;
+import org.jboss.pnc.model.TargetRepository;
+import org.jboss.pnc.model.User;
+import org.jboss.pnc.spi.datastore.predicates.ArtifactPredicates;
+import org.jboss.pnc.spi.datastore.repositories.ArtifactRepository;
+import org.jboss.pnc.spi.datastore.repositories.ProductMilestoneRepository;
+import org.jboss.pnc.spi.datastore.repositories.TargetRepositoryRepository;
+
+import static org.jboss.pnc.constants.ReposiotryIdentifier.DISTRIBUTION_ARCHIVE;
+import static org.jboss.pnc.constants.ReposiotryIdentifier.INDY_MAVEN;
+
+/**
+ *
+ * @author jbrazdil
+ */
+@Transactional
+public class DeliverableAnalyzerResultProcessor {
+    private static final String KOJI_PATH_MAVEN_PREFIX = "/api/content/maven/remote/koji-";
+
+    @Inject
+    private ProductMilestoneRepository milestoneRepository;
+    @Inject
+    private ArtifactRepository artifactRepository;
+    @Inject
+    private TargetRepositoryRepository targetRepositoryRepository;
+
+    /**
+     *
+     * @param milestoneId
+     * @param builds
+     * @param distributionUrl
+     */
+    public void processBuild(int milestoneId, Collection<Build> builds, String distributionUrl, User user) {
```

Comment:
Add javadoc for this method, please.

Reply:
ah, I inteded to add javadoc as you cen see but then somehow forgot about it